apps/openmw/mwmechanics/combat.hpp

```diff
@@ -55,6 +55,9 @@ void applyFatigueLoss(const MWWorld::Ptr& attacker, const MWWorld::Ptr& weapon,
 float getFightDistanceBias(const MWWorld::Ptr& actor1, const MWWorld::Ptr& actor2);
 
 bool isTargetMagicallyHidden(const MWWorld::Ptr& target);
+
+float getAggroDistance(const MWWorld::Ptr& actor, const osg::Vec3f& lhs, const osg::Vec3f& rhs);
```

Comment:
Redundant.

Reply:
No.

What is redundant? Function, declaration or something else? Why? What do you want instead?

Reply:
Sorry, got confused in the intention behind the function. Maybe add a comment which describes its absolute difference from getPathDistance (it's easy to mix them up if you're not careful otherwise).